lib/loggability/http_log_device.rb

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env ruby
+# vim: set nosta noet ts=4 sw=4:
+# encoding: utf-8
+
+require 'socket'
+require 'net/https'
+require 'json'
+require 'concurrent'
+require 'loggability/logger' unless defined?( Loggability::Logger )
+
+require 'loggability/log_device'
+
+# This is the a generalized class that allows its subclasses send
```

Comment:
missing a "to" - "subclasses send to". Also, maybe some specific mention of running on a separate thread?